cmd/api/src/api/v2/database_wipe.go

```diff
@@ -81,7 +81,7 @@ func (s Resources) HandleDatabaseWipe(response http.ResponseWriter, request *htt
 		Model: &model.AuditData{
```

Comment:
GitHub won't let me comment on the exact line, but the `Action` on Line 80 should be constantized as well.